core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -657,7 +667,13 @@ var ExplorationEditorMainTab = function() {
     expect(interactionTab(INTERACTION_ID_TO_TAB_NAME[interactionId])
       .isDisplayed()).toBe(true);
     interactionTab(INTERACTION_ID_TO_TAB_NAME[interactionId]).click();
-    expect(interactionTile(interactionId).isDisplayed()).toBe(true);
+
+    var targetTile = interactionTile(interactionId);
+    waitFor.elementToBeClickable(
+      targetTile,
```

Comment:
is this resolve the issue you have mentioned in the PR explanation?

Reply:
Yes, this solved the issue.